src/core/lib/channel/channel_args.c

```diff
@@ -329,7 +329,8 @@ const grpc_arg *grpc_channel_args_find(const grpc_channel_args *args,
   return NULL;
 }
 
-int grpc_channel_arg_get_integer(grpc_arg *arg, grpc_integer_options options) {
+int grpc_channel_arg_get_integer(const grpc_arg *arg,
+                                 grpc_integer_options options) {
```

Comment:
which one, `options`? It's passed by value so the benefits of const are much smaller.